addons/payment_transfer/migrations/11.0.1.0/noupdate_changes.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+  <record id="payment.payment_acquirer_transfer" model="payment.acquirer">
+    <field name="pre_msg"><![CDATA[
+        <p>Transfer information will be provided after choosing the payment aquirer.</p>]]></field>
+    <field name="view_template_id" ref="transfer_form"/>
+    <field name="auto_confirm"/>
```

Comment:
Same as https://github.com/OCA/OpenUpgrade/pull/1197#pullrequestreview-78895240

Reply:
@StefanRijnhart Thank you, i removed auto_confirm